sdk/nodejs/tests/unwrap.spec.ts

```diff
@@ -244,6 +244,11 @@ describe("unwrap", () => {
             createOutput(Promise.resolve({ a: createOutput(1, r4, r5)}), r1, r2),
             { a: 1 },
             [r1, r2, r4, r5]));
+
+        it("across 'all'", testResources(
+            all([Promise.resolve({ a: createOutput(1, r1, r2)}), Promise.resolve({ b: createOutput(2, r3, r4)})]),
```

Comment:
Can you add some tests that cover this when the inner outputs contain unknown values?

Reply:
yes.

Reply:
Done.